packages/app/src/pages/components/Login.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react'
+import {redirectTo, getCookieValue, setCookie} from '../utils'
+import * as styles from '../index.module.sass'
+
+const HOME = 'http://localhost:8000'
+const SERVER_OUTH_LOGIN = 'http://localhost:3000/oauth/login'
+
+const resetUserCookies = () => {

```

Comment:
Please move this function to the `utils.js` file and pass the cookies to delete as an argument. For example:
```ts
const deleteCookies = (cookiesToDelete: string[]): void  => {
...
}
```